File: SecondYear/chaudharyaryanpanwar_AryanPanwar_2226cs1053_2/Week_4/MaximumConsecutiveOnes.cpp
int traffic(int n, int m, vector<int> vehicle) {
	// Write your code here.
	int left = 0 ; 
	int right = 0 ; 
	int count = 0 ; 
	int maxOnes = 0 ;
        for (int i = 0; i < n; i++) {
			count = 0 ;
          for (int j = i; count<=5 && j<n; j++) {
			  if (vehicle[j]==0)count++;
			  else continue;
			  if ((j-i)<maxOnes)maxOnes = j-i;
		  }
        }
		return maxOnes;
		
}

File: SecondYear/aayushker_AayushkerSingh_2226CSE(AI)1010_2/Week_4/Maximum_Consecutive_Ones.py
def traffic(n: int, m: int, vehicle: [int]) -> int:
    # Write your code here.
    n = len(vehicle)
    left = 0  
    max_length = 0  
    zero_count = 0 

    for right in range(n):
        if vehicle[right] == '0':
            zero_count += 1

        while zero_count > m:
            if vehicle[left] == '0':
                zero_count -= 1
            left += 1

        max_length = max(max_length, right - left + 1)

    return max_length
    pass
